tun/tun_darwin.go

```diff
@@ -213,49 +213,50 @@ func (tun *NativeTun) File() *os.File {
 	return tun.tunFile
 }
 
-func (tun *NativeTun) Events() chan Event {
+func (tun *NativeTun) Events() <-chan Event {
```

Comment:
Seems like a reasonable change, but unrelated to the rest of this patch right? So separate it out.

Reply:
Do you want this in a separate commit as part of this series, or a separate PR?

Reply:
It doesn't matter at all to me. My workflow is just `curl github... | git am` and then checking locally. You can also push to a `jw/for-jason` branch in the repo (which is what Brad usually does).

Reply:
Pushed here https://github.com/tailscale/wireguard-go/commit/f23d9d2050c8b0464d15ff2366d39d99c4dc35a8

GitHub serves my public keys via <username>.keys if you want me to push here or to git.zx2c4.com/wireguard-go.

Reply:
Applied, thanks.

@jwhited @raggi You guys can push branches to git.zx2c4.com:wireguard-go with prefixes of jw/ and jt/, respectively. If you want me to merge something, poke me on Signal or IRC or however you want. If you push to a */for-jason branch, and I happen to see it, I'll merge it, but I don't always see it, so don't hesitate to poke. Larger things that require actual review and back and forth can go through whatever platform is easiest - github review (like this PR), email, annotated pastebin diffs, blurry photographs of napkins, etc -- whichever you prefer.

Reply:
(I imported all your latest keys from github.)